WooCommerce/WooCommerceTests/ViewRelated/Dashboard/Settings/ThemeSettingViewModelTests.swift

```diff
@@ -0,0 +1,52 @@
+import XCTest
+@testable import WooCommerce
+@testable import Yosemite
+
+@MainActor
+final class ThemeSettingViewModelTests: XCTestCase {
+
+    private let sampleSiteID: Int64 = 123
+
+    func test_updateCurrentThemeName_updates_currentThemeName_properly() async {
```

Comment:
nit: I guess we are testing `loadingCurrentTheme` value and maybe we can rename this method to reflect that. 
```suggestion
    func test_updateCurrentThemeName_updates_loadingCurrentTheme_properly() async {
```